main/zoe/api/Zoe-Intro.md

```diff
@@ -0,0 +1,109 @@
+
+
+*Invite*s are ERTP *payments*, so the same principles apply to them as to what you usually think of ERTP payments. You can send the same payment or invite to multiple parties...but the first party to *redeem* it removes the value from the shared payment. As part of redeeming it, they then create their own new payment object with the same amount. Remember, Zoe destroys invites on their redemption.
+
+
+When you receive an *invite* to an *offer*, you should first ensure it is a valid invitation to the type of contract that you want to participate in.  The second thing to check is the `installationHandle` in the invite: this indicates what particular contract code will be used. You can use the `installationHandle` to make sure it’s a contract you’ve used before, and you can also use `zoe.getInstallation(installationHandle)` to have Zoe send you the contract code itself. 
+
+
+If you want to be a party to the contract, *redeem* the invite by providing a *proposal* and a *payment* for the proposal, which Zoe escrows. 
+
+
+*Proposals* consist of:
+- What the user wants (*want key*).
+- What the user offers for what they want (*give key*).
+- How the user can cancel the contract (*exit key*). 
+
+
+The names of the *want* and *give* *keys* are the keywords passed in the keyword record when you created the contract instance. Their values are ERTP amounts of how many of the asset you either want or are willing to give in exchange. 
+Redeeming an offer returns a *seat* at that contract instance and a *payout*. The *seat* object's methods let the user call API commands on the contract instance. (**Note**: we expect this last to change very soon). The *payout* is what the user gets when their offer is completed; either a refund of what they offered or what they wanted to get from the offer. 
+
+
+Once two proposals for a contract match, as determined by the contract instance, the contract instance tells Zoe to *reallocate* the current allocations (allocations = I have 3 brick, you have 2 wool) for those offers, and then completes the offers with Zoe, giving the users their payout. By match we mean that, for example, one party wants three wood and offers two brick for it. The other party wants two brick and offers three wood for it. What they want and what they offer for it sync up.
+
+
+If the offer completes successfully, Zoe creates two new two payments from its escrow, which is stored in Zoe purses. and assigns them to the successful parties. If it doesn't execute successfully, Zoe creates two payments from its escrow and returns them to their original owners.
```

Comment:
> If the offer completes successfully, Zoe creates two new two payments from its escrow, which is stored in Zoe purses. and assigns them to the successful parties. If it doesn't execute successfully, Zoe creates two payments from its escrow and returns them to their original owners.

I think this part should be rewritten, because both cases are the same thing in fact. It should be something like:

"When a contract instance completes an offer, Zoe withdraws the current allocation amounts for that offer from its purses (Zoe was holding the value in escrow), and resolves the user's payout promise with these payment. (Note: actually, it is a promise for a payment, but we explain more here, yada yada). It doesn't matter whether the user is getting a refund or getting what they want - the payout mechanism is the same"